x-pack/plugins/monitoring/server/lib/metrics/__snapshots__/metrics.test.js.snap

```diff
@@ -461,6 +461,55 @@ Object {
     "usageField": "cpuacct.total.ns",
     "uuidField": "beats_stats.beat.uuid",
   },
+  "apm_cgroup_memory": QuotaMetric {
+    "aggs": Object {
+      "periods": Object {
+        "max": Object {
+          "field": "beats_stats.metrics.beat.cgroup.memory.stats.periods",
+        },
+      },
+      "periods_deriv": Object {
+        "derivative": Object {
+          "buckets_path": "periods",
+          "gap_policy": "skip",
+          "unit": "1s",
+        },
+      },
+      "quota": Object {
+        "min": Object {
+          "field": "beats_stats.metrics.beat.cgroup.memory.cfs.quota.us",
```

Comment:
I don't understand the intention with these quota calculations? The memory limit is reported as `beats_stats.metrics.beat.cgroup.memory.mem.limit.bytes`, as mentioned in https://github.com/elastic/kibana/issues/96403#issuecomment-819273799. From where did you get the field names `beats_stats.metrics.beat.cgroup.memory.stats.periods` and `beats_stats.metrics.beat.cgroup.memory.cfs.quota.us`?